milli/src/search/new/tests/proximity_typo.rs

```diff
@@ -68,8 +68,8 @@ fn test_trap_basic() {
         [
             Proximity(
                 Rank {
-                    rank: 8,
-                    max_rank: 8,
+                    rank: 4,
+                    max_rank: 4,
```

Comment:
The ranks change because the maximum distance is lower, which means that we pass from 8 different ranks (distance 1 to 7 + out of range) to 4 (distance 1 to 3 + out of range)